ph-ee-connector-ams-mifos/src/main/java/org/mifos/connector/ams/zeebe/ZeebeeWorkers.java

```diff
@@ -188,6 +194,35 @@ public void setupWorkers() {
                 }
             }).name("release-block").maxJobsActive(workerMaxJobs).open();
 
+            zeebeClient.newWorker().jobType("credit-loan-account").handler((client, job) -> {
+                logWorkerDetails(job);
+                if (isAmsLocalEnabled) {
+                    Map<String, Object> variables = job.getVariablesAsMap();
+                    logger.info(variables.toString());
+                    String fineractTenantId = variables.get("tenantId").toString();
```

Comment:
Use Zeebe variables here instead of hardcoding "tenantId" do this for all the variables

Reply:
done for all the variables.